test/mocha/accessorTokens-1492.js

```diff
@@ -0,0 +1,19 @@
+var UglifyJS = require('../../');
+var assert = require("assert");
+
+describe("Accessor tokens", function() {
+    it("Should fill the token information for accessors (issue #1492)", function() {
+        // location               0         1         2         3         4
+        //                        01234567890123456789012345678901234567890123456789
+        var ast = UglifyJS.parse("var obj = { get latest() { return undefined; } }");
+
+        // test there are no nodes without tokens
+        var checkWalker = new UglifyJS.TreeWalker(function(node, descend) {
+            if (node instanceof UglifyJS.AST_Accessor) {
+                assert.equal(node.start.pos, 22);
+                assert.equal(node.end.endpos, 46);
```

Comment:
I think is is a matter of oppinion. I will try to explain why I think with it should start at 22:

The AST hierarchy is:
`AST_ObjectProperty` contains
- `key` of type `AST_KeySymbolRef`
- `value` of type `AST_Accessor`

As `key` is what contains the property name, I think it makes a sense to let the `AST_Accessor` to start after it. Perhaps its name is a bit misleading and it should be called something like `AST_AccessorBody`, but I doubt you will want to break AST compatibility because of this.